docs/md/routing.md

```diff
@@ -176,6 +176,15 @@ without any tokenization.
 @(has_beginning("The Quick Brown", "quick brown")) → false
 ```
 
+<h2 class="item_title"><a name="test:has_category" href="#test:has_category">has_category(result, categories)</a></h2>
+
+Tests whether the category of a result on of the passed in `categories`
+
+
+```objectivec
+@(has_intent(results.foo, "book_flight", 0.5)) → ERROR
```

Comment:
wrong example here.

Reply:
ah crap forgot about making these examples be more useful..